src/api/start.js

```diff
@@ -76,5 +76,14 @@ export default async (providedOptions = {}) => {
     }
   });
 
+  await new Promise((resolve) => {
```

Comment:
Don't think we should wait for the child to exit here, otherwise returning the child is pointless.  This logic needs to be in the CLI `electron-forge-start.js` file.

Also this file is used as a JS API in other projects, calling process.exit here is a bad plan, we should only call process.exit in our CLI files